packages/dds/tree/package.json

```diff
@@ -73,6 +73,7 @@
     "@types/mocha": "^8.2.2",
     "@types/node": "^14.18.0",
     "@types/random-js": "^1.0.31",
+    "@types/ungap__structured-clone": "^0.3.0",
```

Comment:
Is this change related to the PR's purpose?

Reply:
Yes. This is necessary to cause packages/dds/tree/src/utils.ts to not implicitly use the any type for the import of structured-clone.